README.md

```diff
@@ -4,7 +4,16 @@ CakePHP 3 Codeception Module
 [![Build Status](https://api.travis-ci.org/cakephp/codeception.png)](https://travis-ci.org/cakephp/codeception)
 [![License](https://poser.pugx.org/cakephp/codeception/license.svg)](https://packagist.org/packages/cakephp/codeception)
 
-A [codeception](http://codeception.com) module to test your CakePHP 3 powered application.
+***What?*** A [codeception](http://codeception.com) module to test your CakePHP 3 powered application.
+
+***Why?*** Using Codeception with CakePHP opens up a whole new set of testing capabilities for your app letting you do
+
+- **front-end testing** _(i.e. browser-based workflow tests)_
+  - [x] **[[Functional Tests]](http://codeception.com/docs/05-FunctionalTests)**  - does our Cake App run the way we expect it to
+  - [x] **[[Acceptance Tests]](http://codeception.com/docs/04-AcceptanceTests)**  - does our Cake App do what the customer/user expects it to
+- **back-end testing** _(i.e. direct app function calls)_
+   - [x] **[[API Tests]](http://codeception.com/docs/10-WebServices)** - does my Cake App's API work 
+   - [ ] **[[Unit Tests]](http://codeception.com/docs/06-UnitTests)** - do our App's internal functions do what they should
```

Comment:
@cwbit that is true but using this module adds even more unit testing capabilities to the already solid cake test suite and most importantly, unifies the API for writing tests (across all suites), if you're writing `Cept` or `Cest` unit test cases and/or extending the `Codeception\TestCase`.